extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -941,6 +941,35 @@ describe('amp-a4a', () => {
         });
       });
     });
+    // TODO (keithwrightbos) - move into above e2e once signed creative with
+    // ampImage within creative can be regenerated.
+    it('should prefetch amp images', () => {
+      return createIframePromise().then(fixture => {
+        setupForAdTesting(fixture);
+        fetchMock.getOnce(
+            TEST_URL + '&__amp_source_origin=about%3Asrcdoc', () => adResponse,
+            {name: 'ad'});
+        const doc = fixture.doc;
+        const a4aElement = createA4aElement(doc);
+        const a4a = new MockA4AImpl(a4aElement);
+        sandbox.stub(a4a, 'getAmpAdMetadata_', creative => {
+          const metaData = AmpA4A.prototype.getAmpAdMetadata_(creative);
+          metaData.ampImages = ['https://prefetch.me.com?a=b', 'http://do.not.prefetch.me.com?c=d',
+            'https://prefetch.metoo.com?e=f'];
+          return metaData;
+        });
+        a4a.buildCallback();
+        a4a.onLayoutMeasure();
+        return a4a.layoutCallback().then(() => {
+          expect(doc.querySelector('link[rel=preload]' +
+            '[href="https://prefetch.me.com?a=b"]')).to.be.ok;
```

Comment:
String continuation onto next line should be indented 4 spaces, correct? (I suspect this is a case of the indentation being wrong but the file is omitted by the linter)

Reply:
It passes gulp lint and I am fairly positive it is enforced on test files.